### ISSUE TITLE
CRYPTOCAPITAL SCAM

### ISSUE THREAD

**Issue by frosty00** [ccxt/ccxt#4826]
Do not use www.cryptocapital.co

Exchanges that use cryptocapital:

* bitfinex
* cex.io
* coinapult
* exmo

https://bitcointalk.org/index.php?topic=5118599.0

^ There are many more threads like this

@kroitor maybe we could give exchanges a reputation score too, like https://cer.live/, and use votes from the community to decide it. This could be done with a link that uses github's oauth2 to verify uniqueness and we could store the ratings on our server? Maybe out of 5 stars?

**Comment by frosty00** [comment]
https://github.com/frosty00/ccxt/tree/stars
https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps/

think voting on exchanges would be a cool feature

**Comment by DmitryVil** [comment]
Could you please to write a little bit more about exchanges you mentioned in your first post?
I have some experience of work with exmo and bitfinex but haven't seen any significant problem.
In any case, it is very useful to have another opinion.
Thank you in advance,
Dmitry

**Comment by frosty00** [comment]
I am still waiting on my pending withdrawal from them for over one month. It is not a significant amount of money but purely disrespectful to make someone wait a month and not give them any indications of where there money is except that "intermediary banks are withholding funds".

Will update / remove this post if they give me money.

**Comment by kroitor** [comment]
> @kroitor maybe we could give exchanges a reputation score too, like https://cer.live/, and use votes from the community to decide it. This could be done with a link that uses github's oauth2 to verify uniqueness and we could store the ratings on our server? Maybe out of 5 stars?

I totally agree on this, and I think the idea is great! I suggest that we open a new issue to discuss it, just for the separation of concerns. 

We'll keep this post open for discussion on the CryptoCapital withdrawal problems. This is very important and the issue is critical – users should not risk money until CryptoCapital proves the opposite.

https://bitcointalk.org/index.php?topic=2810707.0

**Comment by npomfret** [comment]
For what it's worth I've used Crypto Capital quite a lot with mixed success.  They do appear to suffer from HUGE delays from time to time, but I've never NOT received my money.  Their support is mixed too, they do sometimes reply and sometimes not.

**Comment by kroitor** [comment]
@npomfret what is the average and what was the longest waiting time for your withdrawals?

**Comment by DmitryVil** [comment]
> I am still waiting on my pending withdrawal from them for over one month. It is not a significant amount of money but purely disrespectful to make someone wait a month and not give them any indications of where there money is except that "intermediary banks are withholding funds".
> 
> Will update / remove this post if they give me money.
> 
> @DmitryVil it only applies if you make the withdrawal via cryptocapital.co
---------------------------------------------------------------------------------------------------------
Thank you very much for the information about cryptocapital.co 
It is very very useful.
Maybe it also will be useful for you - I have had dozens of withdrawal transactions with bitfinex/ethfinex  and exmo, and have never seen any problem - it has been quickly and properly.

Thank you again.
Dmitry

**Comment by kroitor** [comment]
@DmitryVil did you do those transactions via CryptoCapital (to fiat) or those were mainly crypto withdrawals?

**Comment by npomfret** [comment]
I don't have statistics to hand, but I've withdrawn fiat (USD, GBP and EUR) from crypto captial to a UK bank.  In general it takes between a few days and a week.  But at times I seem to remember it has taken many weeks, I'm sorry I don't have more accurate data.

I've not gone from bitfinex to Crypto Capital, didn't even know that was possible.   @DmitryVil ?

**Comment by npomfret** [comment]
Has anyone withdrawn Euros from Crypto Capital recently?  How was your experience?

**Comment by viktorius007** [comment]
I have recently tried to withdrawl USD and have been told it's impossible and there is no timeframe for a solution.  My question extends on npomfret's "Has anyone withdrawn any fiat through CC and if so, what currency was it and how long did it take?"

**Comment by kroitor** [comment]
> My question extends on npomfret's "Has anyone withdrawn any fiat through CC and if so, what currency was it and how long did it take?"

I think @frosty00 has tried withdrawing GBP and Crypto Capital has been holding the money for over a month since then. Still not resolved.

**Comment by npomfret** [comment]
I've just noticed that cex.io are no longer offering Crypto Capital deposits and withdrawals.  I think it's safer to stay away for now.

**Comment by kroitor** [comment]
Updates:

- https://www.coindesk.com/bitfinex-denies-laundering-says-its-a-victim-of-fraud-by-crypto-capital
- https://www.coindesk.com/crypto-capital-principal-indicted-on-fraud-confirming-bitfinex-allegations

**Comment by andrewreedus7** [comment]
“I invested with a Crypto broker without proper research to know what I was hoarding my hard-earned money into, only to find out it was a hoax and sham. I invested over $180,000 USD estimated to be 6.3 BTC. I was unable to make any withdrawals out of my initial deposits let alone the gains he claimed I have earned even after meeting the bogus fees and charges he is always requesting. Fortunately, I got to know about CYBER GENIE's Bitcoin recovery programmers through research and positive reviews on Google. After a couple of hours consulting with them, all my funds were recovered including my profits. I can’t thank these guys enough for making me not another prey to these scammers. Thank you Cyber Genie. Consult them via:
“Cy ber genie@ Cy ber services . com”
“WhatsApp!! +1,252,5,1,2-03,91”
